packages/grafana-ui/src/components/Segment/SegmentInput.story.tsx

```diff
@@ -87,14 +87,14 @@ export const InputWithAutoFocus = () => {
       {inputComponents.map((InputComponent: any, i: number) => (
         <InputComponent initialValue="test" key={i} />
       ))}
-      <a
+      <button
```

Comment:
We'd be careful when changing anything to `button` as it has `submit` type by default. Normally its not a big deal, but when used in forms it will trigger a default submit behavior. Therefore I think we'd add `type='button'` to all the buttons by default (our Button component already has that).

Reply:
Thanks for your comment @Clarity-89 😊 I will add `type='button'`.